components/com_kunena/site/models/user.php

```diff
@@ -151,8 +151,16 @@ public function getItems() {
 			$query .= " ORDER BY {$db->quoteName($this->getState ( 'list.ordering'))} {$this->getState ( 'list.direction')}";
 
 			$db->setQuery ( $query, $limitstart, $limit );
-			$items = $db->loadColumn();
-			KunenaError::checkDatabaseError();
+			try
```

Comment:
Generally its not a good practice to have a try and catch block here. The best place to have try and catch blocks is on the state of the object. When an error gets thrown from the db because of loadColumn the object is in a bad state because the data was never retrieved.  I guess an analogy is your engine to your care is overheating, the engine will keep running until it seizes.  You will be given alerts that your engine is on fire, but the actual error about your engine block failure is not until the car stops working when it is expected to keep working.  This is not a case for every case, however we have a very simplistic call here which really does not need a try catch block for an error on input that other code should be checking.